android/titanium/src/java/org/appcelerator/titanium/TiC.java

```diff
@@ -674,12 +674,18 @@
 
 	/**
 	 * @module.api
+	 * @deprecated as of Android 6.0 this property will be replaced by PROPERTY_ENTER_TRANSITION
+	 * and will be removed in android 7.0
```

Comment:
We talked about SDK 6.0.0 and 7.0.0, not Android 6.0.0 and 7.0.0. We can deprecate it in 6.1.0 and remove in 8.0.0, 7.0.0 (one major release) is too close.